authors/apps/articles/tests/basetests.py

```diff
@@ -538,6 +544,85 @@ def update_comment_succefully(self):
             "articles:commentdetail", args=["this-is-mine", id]),
             data=json.dumps(self.update_comment),
             content_type="application/json")
+#########################################################################################################
```

Comment:
Hello @a-braham ,
Please remove this line. It is not necessary for the functionality of the API